extensions/ql-vscode/test/vscode-tests/no-workspace/data-extensions-editor/model-details/model-details-data-provider.test.ts

```diff
@@ -89,4 +93,45 @@ describe("ModelDetailsDataProvider", () => {
       expect(onDidChangeTreeDataListener).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe("getChildren", () => {
+    const supportedExternalApiUsage = createExternalApiUsage({
+      supported: true,
+    });
+
+    const unsupportedExternalApiUsage = createExternalApiUsage({
+      supported: false,
+    });
+
+    const externalApiUsages: ExternalApiUsage[] = [
+      supportedExternalApiUsage,
+      unsupportedExternalApiUsage,
+    ];
+    const dbItem = mockedObject<DatabaseItem>({
+      getSourceLocationPrefix: () => "test",
+    });
+
+    const usage = createUsage({});
+
+    it("should return [] if item is a usage", async () => {
+      expect(dataProvider.getChildren(usage)).toEqual([]);
+    });
+
+    it("should return usages if item is external api usage", async () => {
+      const externalApiUsage = createExternalApiUsage({ usages: [usage] });
+      expect(dataProvider.getChildren(externalApiUsage)).toEqual([usage]);
+    });
+
+    it("should show all externalApiUsages if hideModeledApis is false and item is undefined", async () => {
+      const hideModeledApis: boolean = false;
```

Comment:
(Minor) you shouldn't need to define the type
```suggestion
      const hideModeledApis = false;
```
Same on L132 and a few places in `model-details-panel.test.ts` - worth doing a search for `const hideModeledApis: boolean =` to find all occurrences.

Reply:
👍